FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -682,13 +686,13 @@ - (void)notifyRefreshedFCMToken {
     return;
   }
   if ([self.delegate respondsToSelector:@selector(messaging:didReceiveRegistrationToken:)]) {
-    [self.delegate messaging:self didReceiveRegistrationToken:self.tokenManager.defaultFCMToken];
+    [self.delegate messaging:self didReceiveRegistrationToken:_tokenManager.defaultFCMToken];
```

Comment:
I wonder why use ivar `_tokenManager` here but a property `self.tokenManager` at other places?

Reply:
You are right, self.tokenManager is better here since there's no specific extra action in getTokenManager.